openfisca/api_config.ini

```diff
@@ -0,0 +1,49 @@
+# OpenFisca-Web-API - Development environment configuration
+#
+# The %(here)s variable will be replaced with the parent directory of this file.
+
+[DEFAULT]
+debug = true
+
+[server:main]
+use = egg:Paste#http
+host = 0.0.0.0
+port = %PORT%
+
+[app:main]
+use = egg:OpenFisca-Web-API
+country_package = openfisca_france
+log_level = DEBUG
+reforms =
+    openfisca_france.reforms.aides_cd93.aides_cd93
+extensions=
+    openfisca_paris
```

Comment:
Probably not by itself. It’ll probably be more surprising for our upcoming new contributors, but we’re still hoping to get the CD93 to commit some more aides, so let’s assign that task to them  ;)